ts/Extensions/DashboardPlugin/DataGridComponent.ts

```diff
@@ -0,0 +1,281 @@
+/* *
+ *
+ *  (c) 2012-2021 Highsoft AS
+ *
+ *  License: www.highcharts.com/license
+ *
+ *  !!!!!!! SOURCE GETS TRANSPILED BY TYPESCRIPT. EDIT TS FILE ONLY. !!!!!!!
+ *
+ *  Authors:
+ *  - Karol Kolodziej
+ *
+ * */
+
+'use strict';
+
+/* *
+ *
+ *  Imports
+ *
+ * */
+
+import type Options from '../../Core/Options';
+import type DataGrid from '../../DataGrid/DataGrid';
+
+import Component from '../../Dashboard/Component/Component.js';
+import DataStore from '../../Data/Stores/DataStore.js';
+import HighchartsSyncHandlers from './HighchartsSyncHandlers.js';
+import U from '../../Core/Utilities.js';
+const {
+    createElement,
+    merge,
+    uniqueKey
+} = U;
+
+
+/* *
+ *
+ *  Class
+ *
+ * */
+
+/**
+ * DataGrid component for the Highsoft Dashboard.
+ * @private
+ * @class
+ * @name Highcharts.DashboardComponent
+ */
+class DataGridComponent extends Component<DataGridComponent.ChartComponentEvents> {
+
+    /* *
+     *
+     *  Static properties
+     *
+     * */
+
+    public static syncHandlers = HighchartsSyncHandlers;
```

Comment:
Question here might be if we should build out these existing handlers to support the datagrid, or have a separate file for the DataGridSyncHandlers 🤔

Reply:
Synch handler is not yet implemented but that is actually a good question! And I think it depends on what we would like to synchronize. But I guess it should be a separate file as you suggested. @bre1470 @pawellysy

Reply:
Separate files sound better IMO